inference-engine/src/gna_plugin/backend/am_intel_dnn.cpp

```diff
@@ -1619,11 +1588,11 @@ void GNAPluginNS::backend::AMIntelDNN::InitGNAStruct(intel_nnet_type_t *ptr_nnet
                     pConvolutionalLayer->nBytesBias = component[i].op.conv1D.num_bytes_per_bias;
                     pConvolutionalLayer->nBytesFilterCoefficient = component[i].op.conv1D.num_bytes_per_weight;
                     pConvolutionalLayer->nFilters = component[i].op.conv1D.num_filters;
-                    pConvolutionalLayer->nFilterRows = comp.op.conv1D.num_filter_coefficients / comp.op.conv1D.num_feature_map_columns;
+                    pConvolutionalLayer->nFilterRows = comp.op.conv1D.num_filter_coefficients / comp.op.conv1D.convStride;
                     pConvolutionalLayer->nFilterCoefficients = component[i].op.conv1D.num_filter_coefficients;
                     pConvolutionalLayer->nFeatureMaps = 1;
-                    pConvolutionalLayer->nFeatureMapRows = component[i].op.conv1D.num_feature_map_rows;
-                    pConvolutionalLayer->nFeatureMapColumns = component[i].op.conv1D.num_feature_map_columns;
+                    pConvolutionalLayer->nFeatureMapColumns = component[i].op.conv1D.convStride;
```

Comment:
Wouldn't it make sense to change the variables naming here as well? Currently "stride" is used to some point and then again "feature map" is used and it's not obvious what's the connection between stride and "feature map columns". Or am I missing something?

Reply:
These are part of GNA Library 1.0 API - we can not change it.
It is not a big problem as will be removed once the support for GNA 1.0 library not needed.

Reply:
OK, thanks for explanation.